.gitignore

```diff
@@ -120,3 +120,9 @@ chromedriver.exe
 
 ### Vagrant ###
 .vagrant
+
+### Terraform ###
+**/.terraform/*
+*.tfstate
+*.tfstate.*
+*.tfvars
```

Comment:
You haven't ignored the `.terraform.lock.hcl` file which is good, but I don't see it checked in. It would be nice to include that [as per the terraform docs](https://www.terraform.io/docs/language/dependency-lock.html) so that the provider versions are strictly versioned between runs

Reply:
from the docs it looks like because  my terraform version is <0.14 (0.13.5), I won't have this. should i upgrade my terraform and include it?

Reply:
Ah, I'm not too worried if you're just on an older version 😄